tests/checkasm/vvc_deblock.c

```diff
@@ -51,7 +52,8 @@ static const uint32_t pixel_mask[3] = {0xffffffff, 0x03ff03ff, 0x0fff0fff};
 #define Q2 buf[2 * xstride]
 #define Q3 buf[3 * xstride]
 
-#define TC25(x) ((tc[x] * 5 + 1) >> 1)
+#define TC_SHIFT(j) ((bit_depth < 10) ? ((tc[j] + (1 << (9 - bit_depth))) >> (10 - bit_depth)) : (tc[j] << (bit_depth - 10)))
+#define TC25(x) ((TC_SHIFT(j) * 5 + 1) >> 1)
```

Comment:
This probably should be ported over to hevc_deblock.c as well?

Reply:
if you check the code, hevc will use the same beta. but vvc did no.
you can use the same beta in vvc code. If all conformance passed, maybe we can use the same beta too